lib/modules/versioning/ubuntu/index.ts

```diff
@@ -6,15 +6,81 @@ export const displayName = 'Ubuntu';
 export const urls = ['https://changelogs.ubuntu.com/meta-release'];
 export const supportsRanges = false;
 
+const distroInfo = `Ubuntu 4.10 "Warty Warthog"
+Ubuntu 5.04 "Hoary Hedgehog"
```

Comment:
Is `Ubuntu` not a bit redundant?

Reply:
well yes, but then again its auto generated.
other than that ill be using the [following](https://github.com/renovatebot/renovate/issues/13645#issuecomment-1065063748).